multinet/db.py

```diff
@@ -656,3 +655,4 @@ def delete_upload_collection(upload_id: str) -> str:
     uploads_db.delete_collection(upload_id)
 
     return upload_id
+# TODO: Refactor the below functions into an `Upload` class
```

Comment:
What does this comment refer to?

Reply:
It refers to the upload functions from Mike V's PR. It must have got split apart from his changes, but it's visible in the final change.

Reply:
I did notice this after I left the comment. Oops.

Reply:
Can you file an issue and link it in the comment?